core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -168,6 +169,8 @@ private[spark] class TaskSetManager(
     t.epoch = epoch
   }
 
+  private val sortedPendingTasks = new AtomicBoolean(false)
```

Comment:
Yes, in current change, I do the ordering when create `TaskSet`, there is no change in `TSM` now. Thanks a lot for suggestion :)